composer.json

```diff
@@ -20,7 +20,7 @@
         "composer/semver": "^1.4",
         "doctrine/annotations": "^1.2",
         "gecko-packages/gecko-php-unit": "^2.0",
-        "sebastian/diff": "^1.4",
+        "php-cs-fixer/diff": "dev-master",
```

Comment:
before merging this PR, we shall make a release of `diff` to not enforce to using non-constant dev-master, with release version, composer will have way better installing option